test/test-network/conf/25-ipv6-neigh-retrans-time-0s.network

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Match]
+Name=test25
+
+[Network]
+IPv6RetransmissionTimeSec=0
```

Comment:
Please always set `IPv6AcceptRA=no` unless it is necessary, otherwise networkd wait for extra timeout, and the test becomes slow.

Reply:
Done